parser_library/src/context/macro_param_data.cpp

```diff
@@ -84,6 +70,21 @@ size_t macro_param_data_composite::size() const { return data_.size(); }
 macro_param_data_composite::macro_param_data_composite(std::vector<macro_data_ptr> value)
     : macro_param_data_component(value.size())
     , data_(move(value))
+{
+    value_.append("(");
+    for (size_t i = 0; i < data_.size(); ++i)
+    {
+        value_.append(data_[i]->get_value());
+        if (i != data_.size() - 1)
+            value_.append(",");
+    }
+    value_.append(")");
+}
```

Comment:
Does this work, especially in the light of the new dynamic type?

Reply:
It does. Nevertheless, a `debugger` type of test was added to verify this.

Reply:
But the behavior seems restricted to this one variable...